### ISSUE TITLE
Activate Group command doesn't work when highlighting the group in the sidepanel

### ISSUE THREAD

**Issue by DragosStefanNecula** [tommyme/vscode-bookmark-x#6]
When highlight the group in the side-panel, the activate group command doesn't work. It only works when it's called by pressing the eye.

I would absolutely love the opportunity to fix this myself, I have already forked it, but I am struggling to run the extension (It runs successfully but it doesn't actually do anything, I don't even have the menu active). Could you guide me on how to run this? I will update if I did figure it out myself.
 
I love this plugin, hope you have a nice day, thanks for reading this :).

**Comment by DragosStefanNecula** [comment]
I have managed to run it!

**Comment by tommyme** [comment]
I guess you're using the beta version of Bookmark X, in this version, I'm using brand new rollup build system

follow the following steps to develop
```shell
# 1. install rollup globally
npm i -g rollup
# 2. build extension for debug
cd location/to/vscode-bookmark-x
rollup -c --sourcemap --forceExit
# 3. click VSCode's debug icon(F5) in `Run and Debug view`
```

**Comment by tommyme** [comment]
> When highlight the group in the side-panel, the activate group command doesn't work. It only works when it's called by pressing the eye.
> 
> I would absolutely love the opportunity to fix this myself, I have already forked it, but I am struggling to run the extension (It runs successfully but it doesn't actually do anything, I don't even have the menu active). Could you guide me on how to run this? I will update if I did figure it out myself.
> 
> I love this plugin, hope you have a nice day, thanks for reading this :).

Thank you for using Bookmark X and interested in it, Currently, It doesn't support activate a group in command palette. The only way to activate a group is to press the eye button on the `group tree view item`

**Comment by tommyme** [comment]
solved in [PR#7](https://github.com/tommyme/vscode-bookmark-x/pull/7)
